core/src/main/java/org/bitcoinj/net/discovery/DnsDiscovery.java

```diff
@@ -96,7 +96,7 @@ public DnsSeedDiscovery(NetworkParameters params, String hostname) {
         }
 
         @Override
-        public List<InetSocketAddress> getPeers(long services, long timeoutValue, TimeUnit timeoutUnit) throws PeerDiscoveryException {
+        public List<InetSocketAddress> getPeers(long services, Duration timeout) throws PeerDiscoveryException {
```

Comment:
Should we provide a @Deprecated @Override for the old signature, rather than have it call the super?

Reply:
Are you aware of the old signature in `PeerDiscovery`?

```
    /**
     * @deprecated use {@link #getPeers(long, Duration)}
     */
    @Deprecated
    default List<InetSocketAddress> getPeers(long services, long timeoutValue, TimeUnit timeoutUnit) throws PeerDiscoveryException {
        return getPeers(services, Duration.ofMillis(timeoutUnit.toMillis(timeoutValue)));
    }
```

Reply:
Sorry, I didn't look closely enough. Yeah, that default method will cause the new method in each subclass to be called.

Reply:
So I assume we can mark this as resolved?